ufo/user_test.py

```diff
@@ -22,6 +30,18 @@
   {'email': 'bar@yahoo.com', 'name': 'bob'},
   {'email': 'baz@gmail.com', 'name': 'mark'}
 ]
+FAKE_DIRECTORY_USER_ARRAY = []
+for x in range(0, len(FAKE_EMAILS_AND_NAMES)):
+  fake_add_user = {}
+  fake_add_user['primaryEmail'] = FAKE_EMAILS_AND_NAMES[x]['email']
+  fake_add_user['name'] = {}
+  fake_add_user['name']['fullName'] = FAKE_EMAILS_AND_NAMES[x]['name']
+  fake_add_user['email'] = FAKE_EMAILS_AND_NAMES[x]['email']
+  fake_add_user['role'] = 'MEMBER'
+  fake_add_user['type'] = 'USER'
+  FAKE_DIRECTORY_USER_ARRAY.append(fake_add_user)
```

Comment:
nit: this line would read much better if "fake_add_user" is "fake_directory_user"
FAKE_DIRECTORY_USER_ARRAY.append(fake_directory_user)